nightly_testing/retrigger_nightlies.py

```diff
@@ -91,6 +91,8 @@ def restart_suite(suite):
     """
     print(f"Restarting {suite}")
     play_command = f"cylc play -q {suite}"
+    if "next-cylc" in suite:
+        play_command = f"export CYLC_VERSION=8-next ; {play_command}"
```

Comment:
```suggestion
    if "next-cylc" in suite:
        play_command = f"export CYLC_VERSION=8-next ; {play_command}"
    else:
        play_command = f"cylc play -q {suite}"
```
Using this if-else statement will save calls being made back and forth to memory when a large number of suites need to be restarted as the algorithm only needs to make one assignment statement as opposed to an assignment statement and a call back to the same address in memory to change the play_command if next-cylc is in the suite variable.